static/app/components/structuredEventData/index.tsx

```diff
@@ -380,3 +398,10 @@ const ValueNumber = styled('span')`
 const ValueObjectKey = styled('span')`
   color: var(--prism-keyword);
 `;
+
+const StyledCopyButton = styled(CopyToClipboardButton)`
+  position: absolute;
+  right: ${space(1.5)};
+  z-index: 2;
```

Comment:
You should be able to avoid using `z-index` if you render this after the `<StucturedData />`